keyboards/hs60/v2/keymaps/stanrc85/rules.mk

```
@@ -1,6 +1,6 @@
 # project specific files
 SRC =	keyboards/zeal60/zeal60.c \
-		keyboards/zeal60/rgb_backlight.c \
+    keyboards/zeal60/rgb_backlight.c \
```

Could this be completely reverted, so the PR doesn't touch the file? 

```suggestion
		keyboards/zeal60/rgb_backlight.c \
```